<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-06-19T10:43:09 IST" name="Suite" finished-at="2020-06-19T10:43:11 IST" duration-ms="2290">
    <groups>
    </groups>
    <test started-at="2020-06-19T10:43:09 IST" name="bankmagerlogin" finished-at="2020-06-19T10:43:11 IST" duration-ms="2205">
      <class name="Testcase.bankmagerlogin">
        <test-method signature="bankmager()[pri:0, instance:Testcase.bankmagerlogin@56a6d5a6]" started-at="2020-06-19T10:43:09 IST" name="bankmager" finished-at="2020-06-19T10:43:11 IST" duration-ms="2154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bankmager -->
      </class> <!-- Testcase.bankmagerlogin -->
    </test> <!-- bankmagerlogin -->
    <test started-at="2020-06-19T10:43:11 IST" name="Addcustomertest" finished-at="2020-06-19T10:43:11 IST" duration-ms="19">
      <class name="Testcase.Addcustomertest">
        <test-method is-config="true" signature="setup()[pri:0, instance:Testcase.Addcustomertest@65466a6a]" started-at="2020-06-19T10:42:58 IST" name="setup" finished-at="2020-06-19T10:43:09 IST" duration-ms="10377" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Addcustomer(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Testcase.Addcustomertest@65466a6a]" started-at="2020-06-19T10:43:11 IST" name="Addcustomer" data-provider="Input1" finished-at="2020-06-19T10:43:11 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:52)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:148)
at org.testng.internal.Parameters.handleParameters(Parameters.java:797)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:788)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at Utility.Datadriver.getrowcount(Datadriver.java:36)
at Testcase.Addcustomertest.input1feed(Addcustomertest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:79)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:48)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Addcustomer -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Testcase.Addcustomertest@65466a6a]" started-at="2020-06-19T10:43:11 IST" name="teardown" finished-at="2020-06-19T10:43:12 IST" duration-ms="731" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Testcase.Addcustomertest -->
    </test> <!-- Addcustomertest -->
  </suite> <!-- Suite -->
</testng-results>
